group_vars/postgresql/production.yml

```diff
@@ -34,17 +34,15 @@ postgres_port: 5432
 postgres_admin_user: "{{ postgresadmin }}"
 
 datadog_api_key: "{{ vault_datadog_key }}"
-datadog_service_name: postgresql
-datadog_environment: prod
 datadog_config:
   log_enabled: true
-  process_enabled: true
-datadog_typed_checks:
```

Comment:
@kayiwa why did you remove the typed check?  This is the newer and more correct configuration for datadog

Reply:
Oh, I didn't realize this @carolyncole I am still debugging this. OK for this to go in? if I make it an issue.